package.json

```diff
@@ -30,7 +30,7 @@
     "applicationinsights": "^1.8.7",
     "applicationinsights-native-metrics": "0.0.5",
     "audit-ci": "^3.1.1",
-    "axios": "^0.19.0",
+    "axios": "^0.21.1",
```

Comment:
there is a possible breaking changed that was introduced around this where a parameter is no longer supported for posts. Have slacked you regarding this

Reply:
Looks like just using GETS and specifying the whole url so should be fine